pyk/src/pyk/kcfg/kcfg.py

```diff
@@ -264,18 +264,36 @@ class MergedEdge(EdgeLike):
         edges: tuple[KCFG.Edge, ...]
 
         def to_dict(self) -> dict[str, Any]:
+
+            def _merged_edge_to_dict(edge: KCFG.Edge) -> dict[str, Any]:
```

Comment:
```suggestion
            def merged_edge_to_dict(edge: KCFG.Edge) -> dict[str, Any]:
```

(The functions is only visible from `to_dict` anyways.)